modules/platforms/dotnet/Apache.Ignite.Core/Impl/Client/Services/ServicesClient.cs

```diff
@@ -35,6 +40,9 @@ namespace Apache.Ignite.Core.Impl.Client.Services
     /// </summary>
     internal class ServicesClient : IServicesClient
     {
+        /** Max service topology update period in millis. */
+        internal static readonly TimeSpan SrvTopUpdatePeriod = TimeSpan.FromSeconds(10);
```

Comment:
10 seconds is a bit too short. How often do we expect the topology to change?

Reply:
Java thin client has this period. But we can extend them both to one minute probably. WDYT?

Reply:
Let's extend to a minute and maybe file tickets to make those configurable?

Reply:
1 minute is ok to me. But I dont think anyone is going to configure there tiemouts. Do we need any ticket? A minor improvemets?

Reply:
Btw. Service topology is not updated if client doesn't call the service. This timeout works only with the traffic.

Reply:
My point is, we should optimize for a common use case, which is a stable topology. Requesting new assignment every 10 seconds seems wasteful. One minute is reasonable.

> I dont think anyone is going to configure these tiemouts.
Ok, I agree. Let's keep it hardcoded, we can always make it configurable if users ask for that.